<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuperUser</class>
 <widget class="QWidget" name="SuperUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TextScroll" name="upLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>180</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>180</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="homePage">
      <property name="styleSheet">
       <string notr="true">QWidget{
background:rgba(255,255,255,1);
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnAddClass">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>70</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
    background-color:rgba(255, 255 , 255,70%);
	font: bold 20px;
	border:3px solid red;
    border-radius:8px;
}

QPushButton:hover
{
	background-color:rgba(255, 0 , 0,5%);
    color:rgb(0 , 0 , 255);
}

QPushButton:pressed
{
    color:rgb(0 , 109 , 176);
}</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="text">
          <string>1.添加新教室</string>
         </property>
         <property name="autoRepeatDelay">
          <number>300</number>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btnDelClass">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
    background-color:rgba(255, 255 , 255,70%);
	font: bold 20px;
	border:3px solid red;
    border-radius:8px;
}

QPushButton:hover
{
	background-color:rgba(255, 0 , 0,5%);
    color:rgb(0 , 0 , 255);
}

QPushButton:pressed
{
    color:rgb(0 , 109 , 176);
}</string>
         </property>
         <property name="text">
          <string>2.删除教室</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btnSetTimetable">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
    background-color:rgba(255, 255 , 255,70%);
	font: bold 20px;
	border:3px solid red;
    border-radius:8px;
}

QPushButton:hover
{
	background-color:rgba(255, 0 , 0,5%);
    color:rgb(0 , 0 , 255);
}

QPushButton:pressed
{
    color:rgb(0 , 109 , 176);
}</string>
         </property>
         <property name="text">
          <string>3.设置课表</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="btnFindTimetable">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
    background-color:rgba(255, 255 , 255,70%);
	font: bold 20px;
	border:3px solid red;
    border-radius:8px;
}

QPushButton:hover
{
	background-color:rgba(255, 0 , 0,5%);
    color:rgb(0 , 0 , 255);
}

QPushButton:pressed
{
    color:rgb(0 , 109 , 176);
}</string>
         </property>
         <property name="text">
          <string>4.查询教室租用情况</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="addClassPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QTableView" name="tableView">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>771</width>
            <height>281</height>
           </rect>
          </property>
         </widget>
         <widget class="QFrame" name="frame">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>771</width>
            <height>71</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>10</y>
             <width>130</width>
             <height>30</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>添加教室名称</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>10</y>
             <width>130</width>
             <height>30</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>30</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>620</x>
             <y>10</y>
             <width>130</width>
             <height>30</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnhome">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>130</width>
             <height>30</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>首页</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="delClassPage">
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>791</width>
         <height>391</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="frame_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>781</width>
          <height>51</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="btnhome_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>130</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>首页</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>10</y>
           <width>130</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>删除教室名称</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>10</y>
           <width>130</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>640</x>
           <y>10</y>
           <width>130</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </widget>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>771</width>
          <height>301</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView{
        border:none;
        background:white;
}
QTableView::item:selected{
        color:white;
        background:rgb(34, 175, 75);    
}</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="setTimetablePage">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>791</width>
         <height>391</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="frame_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>771</width>
          <height>61</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="btnhome_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>130</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>首页</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>20</y>
           <width>72</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>请输入教室</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>10</y>
           <width>130</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>640</x>
           <y>10</y>
           <width>130</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </widget>
       <widget class="QTableView" name="tableView_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>181</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>80</y>
          <width>571</width>
          <height>301</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="findTimetablePage">
      <property name="styleSheet">
       <string notr="true">#findTimetablePage{
background:rgba(255,255,255,1);
}</string>
      </property>
      <widget class="QFrame" name="frame_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>781</width>
         <height>381</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="frame_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>771</width>
          <height>61</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="btnhome_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>130</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>首页</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>20</y>
           <width>72</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>请输入教室</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>10</y>
           <width>130</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>640</x>
           <y>10</y>
           <width>130</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </widget>
       <widget class="QTableView" name="tableView_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>761</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextScroll</class>
   <extends>QLabel</extends>
   <header location="global">TextScroll.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
